bionic/aip/task.py

```diff
@@ -101,16 +99,13 @@ def _ai_platform_job_spec(self):
             output["trainingInput"]["workerType"] = self.resource.worker_machine
             output["trainingInput"]["workerConfig"] = {"imageUri": self.job.image}
 
-        if self.tune is not None:
-            output["trainingInput"]["hyperparameters"] = self.tune.spec()
-
         return output
 
     def submit(self) -> Future:
         self._stage()
         spec = self._ai_platform_job_spec()
 
-        if os.environ.get("TEST", "").lower() == "true":
+        if "PYTEST_CURRENT_TEST" in os.environ:
```

Comment:
I'm a little confused at how these tests are intended to work. If we're substituting a `_TestFuture` here, are we still hitting AIP in our tests?

Also, as I noted [here](https://github.com/square/bionic/pull/249/files#r486491423), I'd much rather avoid this kind of "are we in a test" check altogether.

Reply:
Right, these tests don't hit AIP. It runs the task in a subprocess instead. I had to change this condition to actually run the test on AIP.

Re "are we in a test" check, this is part of my second commit (tagged with fixup) and I'm going to remove that commit altogether once we fix this in the base PR (the one with the linked comment).